package.json

```diff
@@ -103,7 +120,11 @@
     "start-server-and-test": "1.9.0",
     "stylelint": "9.10.1",
     "stylelint-config-standard": "17.0.0",
-    "stylelint-order": "0.8.1"
+    "stylelint-order": "0.8.1",
+    "typescript": "3.4.5"
+  },
+  "resolutions": {
+    "@types/react": "16.8.16"
```

Comment:
Yeah. I couldn't update to a recent version either and I know good ol Phil has the same problem.